pilot/pkg/networking/core/listener_inbound.go

```diff
@@ -177,6 +177,13 @@ func (lb *ListenerBuilder) buildInboundHBONEListeners() []*listener.Listener {
 		}
 		l.FilterChains = append(l.FilterChains, chains...)
 	}
+	// If there are no filter chains, populate a dummy one that never matches. Envoy doesn't allow no chains, but removing the
```

Comment:
Don't we usually add a "blackhole" chain as the default? Can we still have that as the final on_no_match?

Reply:
Oh good point

Reply:
Updated